adafruit_platformdetect/board.py

```diff
@@ -117,7 +118,7 @@ def id(self):
             board_id = self._clockwork_pi_id()
         elif chip_id == chips.RK3308:
             board_id = self._rock_pi_id()
-        elif chip_id == chips.RYZEN_V1605B:
+        elif self.board_asset_tag == "SC40-2000-0000-C0|C":
```

Comment:
Can we move the actual tag into the constants/board.py file? You could make an IDs tuple like we did for other boards. That way if UDOO adds additional IDs, it's easy to add new ones in and it looks cleaner.

Reply:
Like this?`
'_UDOO_BOLT_ASSET = "SC40-2000-0000-C0|C"

_UDOO_IDS = (
   UDOO_BOLT,
   _UDOO_BOLT_ASSET,
)'

Reply:
Not quite. More like:
```
_UDOO_BOARD_IDS = (
    UDOO_BOLT: (
        "SC40-2000-0000-C0|C",
    )
)'
```